packages/type/src/utils.ts

```diff
@@ -9,6 +9,7 @@
  */
 
 import { stringify, v4 } from 'uuid';
+import { init } from '@paralleldrive/cuid2';
```

Comment:
Is this a big dependency? Can we move it to optional dependency, so it doesn't come automatically with deepkit/type?

Reply:
It's only 127 lines: https://github.com/paralleldrive/cuid2/blob/main/src/index.js
but it depends on @noble/hashes.
It's total install size is 750kb: https://packagephobia.com/result?p=%40paralleldrive%2Fcuid2

I can move it to peer deps if you prefer